src/book.js

```diff
@@ -0,0 +1,12 @@
+export default function book(){
+    return `
+        <h1>Book</h1>
+            <ol>
+                <li><img src = "" alt = ""></li>
+                <li>Author</li>
+                <li>Price</li>
+                <li>Selling Points</li>
```

Comment:
The _selling points_ should be a list, but the rest of these things deserve their own elements (perhaps `p`, perhaps `div`s... but not `li`)